nix/monomer.nix

```diff
@@ -0,0 +1,53 @@
+{ system, version, flake-utils, ... }:
+self: super:
+with self;
+with haskell.lib;
+with flake-utils.lib;
+with super.haskellPackages.extend (self: super:
+  with haskellPackages; rec {
+    # nanovg = dontCheck (callHackage "nanovg" "0.8.0.0" {
```

Comment:
`nanovg-0.8.0.0` is broken on Nix

Reply:
If using the Hackage version is not possible, we should use the GitHub commit for the time being.

Reply:
we're good here then; resolved